draft-ietf-quic-transport.md

```diff
@@ -2527,6 +2546,26 @@ Reordering might cause frames to be received after closing, see
 {{state-hc-remote}}.
 
 
+## Solicited State Transitions
+
+If an endpoint is no longer interested in the data being received, it MAY send a
+STOP_SENDING frame on a stream in the "open" or "half-closed (local)" state to
+request closure of the stream in the opposite direction.  This typically
+indicates that the receiving application is no longer reading from the stream,
+but is not a guarantee that incoming data will be ignored.
+
+STREAM frames received after sending STOP_SENDING are still counted toward the
+connection and stream flow-control windows.  Even though these frames will be
+discarded, because they are sent before their sender receives the STOP_SENDING,
+the sender will consider the frames to count against its flow-control windows.
+
+Upon receipt of a STOP_SENDING frame on a stream in the "open" state, an
```

Comment:
Good point on hc-remote -- I'll update that.  The point of the paragraph is that if you're currently transmitting data, you SHOULD cut it off.  If you're done transmitting data, but you might still have retransmissions going, you MAY reset the stream or just finish your retransmissions as normal, whichever is easier.

The QUIC_RECEIVED_RST error code is there to make it easier to distinguish RST-because-error from RST-because-you-said-to.